erpnext/accounts/report/sales_register_by_item/sales_register_by_item.py

```diff
@@ -323,116 +307,14 @@ def get_grand_total(filters, doctype):
 def get_deducted_taxes():
 	return frappe.db.sql_list("select name from `tabPurchase Taxes and Charges` where add_deduct_tax = 'Deduct'")
 
-def get_tax_accounts(item_list, columns, company_currency,
```

Comment:
This function is imported in `item_wise_purchase_register.py`

Reply:
will revert it!